openstack/main.tf

```diff
@@ -102,6 +102,9 @@ variable cloudflare_record_texts {
   default = ["*"]
 }
 
+# Provider
+provider "openstack" {}
```

Comment:
what is this for?

Reply:
It is for storing openstack provider settings - currently we don't have one because we don't have speciffic settings - but I think it looks more complete to have the provider there although it is empty. When it is time to publish a stable we might want to put a version tag here as well as the other providers.